pkg/types/reference.go

```diff
@@ -39,7 +47,7 @@ func ParseReference(text string) (*Reference, error) {
 	if len(matches) != 4 {
 		return nil, errors.Errorf("%q is not a valid GetEnvoy reference. Expected format: <flavor>:<version>[/<platform>]", text)
 	}
-	return &Reference{strings.ToLower(matches[1]), strings.ToLower(matches[2]), strings.ToLower(matches[3])}, nil
```

Comment:
this was the only place where we wrote the value missing the hyphen conversion I noticed.